lib/ember-cli/app.rb

```diff
@@ -95,10 +95,14 @@ def ember_path
 
     private
 
-    delegate :match_version?, :non_production?, to: Helpers
+    delegate :match_version?, to: Helpers
     delegate :configuration, to: EmberCLI
     delegate :tee_path, :npm_path, :bundler_path, to: :configuration
 
+    def non_production?
+      !Rails.env.production?
```

Comment:
Could we expose these are configurations, with the default value set from the logic in `Helpers`? 

The project that is driving these changes has `config.consider_all_requests_local = false`